.github/workflows/ci-cd.yml

```
@@ -0,0 +1,71 @@
+
+      - name: Build the project
+        id: build
+        run: pnpm build
+
+      - name: Lint the project
+        id: lint
+        run: pnpm lint
+
+      - name: Comment on PR for build success
+        if: ${{ github.event_name == 'pull_request' && success() }}
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)